utils/frame/frame-utilities-cli/Cargo.toml

```diff
@@ -17,8 +17,21 @@ structopt = "0.3.8"
 codec = { version = "1.3.0", package = "parity-scale-codec" }
 cli-utils = { path = "../../cli-utils", version = "2.0.0-dev" }
 hex = "0.4.2"
+serde = "1.0.106"
+hyper = "0.12.35"
+libp2p = "0.18.1"
+bip39 = "0.6.0-beta.1"
+jsonrpc-core-client = { version = "14.0.3", features = ["http"] }
+sc-rpc = { version = "2.0.0-dev", path = "../../../client/rpc" }
+sc-service = { version = "0.8.0-dev", default-features = false, path = "../../../client/service" }
+sc-client-api = { version = "2.0.0-dev", default-features = false, path = "../../../client/api" }
+futures01 = { package = "futures", version = "0.1.29" }
```

Comment:
so we settled on the futures01 thing? Can't used newer futures in compat mode?

Reply:
unfortunately, this call requires a tokio v0.1 runtime.

```rust
http::connect(&url)
    .and_then(|client: AuthorClient<H, H>| {
        client.insert_key(key_type, suri, public).map(|_| ())
    })
    .map_err(|e| {
        println!("Error inserting key: {:?}", e);
    })
```

Reply:
It sounds like an oversight on our side, but one we can fix later- that you'd need a f01 runtime to use the rpc client...

Reply:
`futures` in the right version are re-exported from `jsonrpc_core`, so you can just do `use jsonprpc_core::futures::Future` instead.